src/components/UserCard.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div class="student" :class="themeClass">
```

Comment:
Right now, there is an `error` in console because you are referencing `themeClass` here, but it's not defined in the `script` section either as a data or as a computed property
![Screen Shot 2019-09-13 at 1 45 58 PM](https://user-images.githubusercontent.com/11170707/64886919-e3302880-d62c-11e9-9379-4ab9e088559b.png)